README.md

```diff
@@ -119,6 +119,10 @@ An array of contributors, with properties such as:
 -   `github`: the GitHub account of the contributor, with or without an `@`.
 -   `twitter`: the Twitter account of the contributor, with or without an `@`.
 
+### `options.appendIfMissing`
+
+Optional. Defaults to **false**. If set to `true` it will append the table if no `Contributors` section header is found.
```

Comment:
More precisely, "it will append a section (that is, a heading and a table) if no `Contributors` heading is found."

Also, this paragraph should be updated:

https://github.com/hughsk/remark-contributors/blob/6c52436664b48d55891fd4f382bc22a5b226b6d6/README.md#L72-L74

Reply:
Updated